src/main/webapp/help/scanSpeed.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    This option will optimize depth of the scan, and it will influence the results and the speed of the scan. For example, if you are on an early stage of development, you can choose the  simple option and get faster results, including the low hanging fruit security issues in the code.
```

Comment:
This help text is Itay's suggestion. There is no help text in AppScan Go. You can either go ahead and get it reviewed by Matt or Jen.

Reply:
Okay, sure

Reply:
Can you run this by Jen?

Reply:
Okay Sure, Matt